src/steps/ExitPoint.js

```diff
@@ -1,15 +1,43 @@
 const Page = require('./Page');
 const destroySession = require('../session/destroySession');
 const { stopHere } = require('../flow');
+const deepEqual = require('deep-equal');
+
+storeStateInUrl = (req, res, next) => {
+  if (!deepEqual(req.query, req.currentStep.values())) {
+    const params = Object.entries(req.currentStep.values())
+      .map(([key, val]) => `${key}=${val}`)
```

Comment:
is there a library for doing this for us? Feels like it should probably include some url escaping / injection protection